pycbc/filter/qtransform.py

```diff
@@ -337,20 +435,29 @@ def qtransform(fseries, Q, f0):
     # apply window to fft
     # normalize and generate bi-square window
     norm = np.sqrt(315. * qprime / (128. * f0))
-    windowed = fseries[start:end].numpy() * (bisquare(window_size) * norm)
+    windowed = fseries[start:end] * (bisquare(window_size) * norm)
+
+    # choice of output sampling rate
+    output_sampling = sampling # Can lower this to highest bandwidth
+    output_samples = dur * output_sampling
 
     # pad data, move negative frequencies to the end, and IFFT
     padded = np.pad(windowed, padding(window_size, output_samples), mode='constant')
     wenergy = npfft.ifftshift(padded)
 
     # return a 'TimeSeries'
     wenergy = FrequencySeries(wenergy, delta_f=1./dur)
-    cenergy = TimeSeries(zeros(output_samples, dtype=np.complex128),
+    tdenergy = TimeSeries(zeros(int(output_samples), dtype=np.complex128),
```

Comment:
Unfortunately, I don't quite remember what the issue was before. But I could do a test run with the old version and see if anything fails.